content/en/docs/concepts/containers/images.md

```diff
@@ -157,6 +157,30 @@ that Kubernetes will keep trying to pull the image, with an increasing back-off
 Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
 which is 300 seconds (5 minutes).
 
+## Serial and parallel image pulls
+
+By default, kubelet pulls images serially. In other words, kubelet will send only
+one image pull request to the image service at a time. Other image pull requests
+have to wait until the one being processed is complete.
```

Comment:
Is this right:
```suggestion
By default, kubelet pulls images serially. In other words, kubelet will send only
one image pull request to the image service at a time. Other image pull requests
have to wait until the one being processed is complete.

If two or more Pods on a node use the same image, the kubelet only makes one image
pull request on behalf of the local Pods. The kubelet on a given node does not coordinate
image pulls with other nodes in your cluster: different nodes can always start image pulls from
the same registry in parallel.

```

Reply:
> For a given container image, the kubelet will send only one image pull request at a time.

This is not only for one given container image. Kubelet sends only one pull request across all pods on the node.